tests/test_update_bug_list.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+import oar.cli.cmd_update_bug_list as cmd_update_bug_list
+from click.testing import CliRunner
+
+class TestBugList(unittest.TestCase):
```

Comment:
@ming1013 if you want to add a test class for CLI, suggest to rename the file with cli not specific cmd name. then you can put other cli related test cases in this file. it does not make sense to create test class for every cli cmd

Reply:
BTW, this class is optional, for cli level testing, it should be e2e test, not unit test. we just need to run it with different options and verify the output is expected or not

Reply:
Thanks @rioliu-rh. I've done e2e test. I'll update in description. Shall I keep this unit test (rename the file) or just remove it?

Reply:
> Thanks @rioliu-rh. I've done e2e test. I'll update in description. Shall I keep this unit test (rename the file) or just remove it?

I think we don't need it. because even we have e2e test suite, we don't any dummy errata resources to run.

Reply:
Thanks, Rio. Removed tests/test_update_bug_list.py